terraform/ipfs-server/process-stream.sh

```diff
@@ -88,7 +87,7 @@ while true; do
       echo "#EXT-X-VERSION:3" >> current.m3u8
       echo "#EXT-X-TARGETDURATION:${HLS_TIME}" >> current.m3u8
       echo "#EXT-X-MEDIA-SEQUENCE:${sequence}" >> current.m3u8
-      tail -n ${M3U8_SIZE} ~/process-stream.log | awk '{print "#EXTINF:"$5",\n'${IPFS_GATEWAY}'/ipfs/"$2}' >> current.m3u8
+      tail -n ${M3U8_SIZE} ~/process-stream.log | awk '{print $6"#EXTINF:"$5",\n'${IPFS_GATEWAY}'/ipfs/"$2}' | sed 's/#EXT-X-DISCONTINUITY#/#EXT-X-DISCONTINUITY\n#/' >> current.m3u8
```

Comment:
I think the sed needs a `g` at the end in case there are multiple discontinuity tags

Reply:
Ahhh good call